### PR TITLE
Move theming support from Beta to Main

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #128 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Move theming support for the auro-checkbox component from beta to main, enabling customization through design tokens and updating documentation to reflect these changes.

New Features:
- Introduce theming support for the auro-checkbox component, allowing customization through design tokens.

Enhancements:
- Refactor auro-checkbox and auro-checkbox-group components to utilize new theming capabilities with tier 3 tokens.

Documentation:
- Update documentation to include instructions on how to restyle components using the new theming support.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
...
+
+* **color:** resolve color theming bugs [#128](https://github.com/AlaskaAirlines/auro-checkbox/issues/128) ([15a433e](https://github.com/AlaskaAirlines/auro-checkbox/commit/15a433e376277fcd0b794c59b32ebe2670f6e271))
+
+# [3.1.0-beta.1](https://github.com/AlaskaAirlines/auro-checkbox/compare/v3.0.1...v3.1.0-beta.1) (2024-07-20)
+
+
+### Features
+
+* add tier 3 tokens for theming capabilities [#128](https://github.com/AlaskaAirlines/auro-checkbox/issues/128) ([1a8246e](https://github.com/AlaskaAirlines/auro-checkbox/commit/1a8246e19e1e975d282fb01a81175ce02394e3f8))
+
```
Comment (jason-capsule42) [inline CHANGELOG.md:16]: We should be removing all "beta" change records prior to merging to main.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]
What is going on with the different background colors here?

Screenshot 2024-09-17 at 11 24 23 AM

**Comment by jordanjones243** [comment]
> What is going on with the different background colors here?
> 
> Screenshot 2024-09-17 at 11 24 23 AM

Updated Figma doc:

Screen Shot 2024-09-17 at 11 42 18 AM

File: CHANGELOG.md (hunk 1 above)
Comment (jordanjones243) [reply CHANGELOG.md]: Issue should be resolved.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**